pkg/grizzly/workflow.go

```diff
@@ -103,7 +103,7 @@ func ListRemote(opts Opts) error {
 	return w.Flush()
 }
 
-// Pulls remote resources
+// Pull pulls remote resources, if opts.JSONSpec is true, saves them as JSON files
```

Comment:
I'd clarify it just save the spec, otherwise it may seem that it's only the wire format what changes.